ilp-core-codecs/src/main/java/org/interledger/core/asn/framework/InterledgerCodecContextFactory.java

```diff
@@ -59,7 +59,18 @@
    * @return A new instance of {@link CodecContext}.
    */
   public static CodecContext oer() {
-    return CodecContextFactory.getContext(CodecContextFactory.OCTET_ENCODING_RULES)
+    final CodecContext ilpCodecContext = CodecContextFactory.oer();
+    return register(ilpCodecContext);
+  }
+
+  /**
+   * Register the ILP codecs into the provided context.
+   *
+   * @param context the context to register the codecs into
+   * @return The supplied {@code context} with ILP Codecs registered into it.
+   */
+  public static CodecContext register(CodecContext context) {
+    return context
```

Comment:
This project generally enforces two style rules. First, method params should always be non-null (see [here](https://github.com/google/guava/wiki/UsingAndAvoidingNullExplained) for why), and if `null` is encountered, the  program should halt. 

Second, method parameters should be `final` where possible to help reduce errors (these styles aren't everywhere in this codebase, but should be). So, something like this:

```java
public static CodecContext register(final CodecContext context) {
    Objects.requireNonNull(context, "context must not be null");
   
    ...
}
```